encrypted-media-respec.html

```diff
@@ -3023,17 +3027,17 @@
         <ul>
           <li>
             <p>
-              Access network resources, either local or remote, except via the user agent, or as part of the user agent, as explicitly permitted by this specification.
+              Access network resources, either local or remote, except via the user agent as explicitly permitted by this specification.
```

Comment:
Most changes in this PR look fine, but I disagree with removing "as part of the user agent".  That text is necessary for completeness.  The goal of this section is to provide an overview of the implementation requirements for security and privacy, specifically where sandboxing is one implementation approach.  The implementation note (which this PR moves back to the CDM definition) is the sole area where this option is stated, and if moved will be remote from this overview paragraph.  I believe moving it back by itself is acceptable, but the "as part of the user agent" text really be maintained for completeness.  Without it, this implementation option may be clear to members of the working group, but it's hardly clear to people reading the spec for the first time.